dynamiq/memory/config.py

```diff
@@ -0,0 +1,6 @@
+from pydantic import BaseModel, Field
+
+
+class Config(BaseModel):
```

Comment:
What do you think about renaming `Config` to `MemoryConfig`? Or maybe even removing this config and making `search_limit` and `search_filters` attributes of the Memory class?

It took me some time to understand what `Config` refers to here. Initially, I thought it was related to Node parameters for the workflow.

Reply:
It was adjusted by @delamainer previously we have MemoryConfig

Reply:
What do you think about moving the parameters of Config into the Memory class as attributes? 
Or do you think it's more beneficial to keep them in a separate config?

Reply:
It would be more beneficial for me to keep them in memory. Let's also move forward with that.